### PR TITLE
[Rl-Worker]: Added Nvidia Device plugin

### PR DESCRIPTION
**Pull request by Ayukha**
Deploys Nvidia device plugin for GPU workers

### CODE REVIEW

**Comment by RishabhJain2018** [comment]
@Ayukha Is this ready to be reviewed?

**Comment by Ayukha** [comment]
@RishabhJain2018 Not yet, I need to test it in the staging server with the current merged code.

**Comment by RishabhJain2018** [comment]
okay, let me know when it is.

**Comment by Ayukha** [comment]
@RishabhJain2018  Please review

File: scripts/workers/rl_submission_worker.py [hunk 1]
```diff
@@ -247,6 +250,27 @@ def update_failed_jobs_and_send_logs(
...
 
+def install_gpu_drivers(api_instance):
+    """Function to get the status of a running job on AWS EKS cluster
+    Arguments:
+        api_instance {[AWS EKS API object]} -- API object for creating deamonset
+    """
+    logging.info("Install GPU Drivers.")
+    link = "https://raw.githubusercontent.com/NVIDIA/k8s-device-plugin/v1.11/nvidia-device-plugin.yml"  # pylint: disable=line-too-long
+    logging.info("Using daemonset file: %s", link)
+    f = urllib.urlopen(link)
```
Comment (RishabhJain2018) [inline scripts/workers/rl_submission_worker.py:261]: Can you please change `f` to something meaningful?

File: scripts/workers/rl_submission_worker.py [hunk 2]
```diff
@@ -247,6 +250,27 @@ def update_failed_jobs_and_send_logs(
                         )
 
 
+def install_gpu_drivers(api_instance):
+    """Function to get the status of a running job on AWS EKS cluster
+    Arguments:
+        api_instance {[AWS EKS API object]} -- API object for creating deamonset
+    """
+    logging.info("Install GPU Drivers.")
```
Comment (RishabhJain2018) [inline scripts/workers/rl_submission_worker.py:258]: `logging.info("Install GPU Drivers.")` -> `logging.info("Installing Nvidia-GPU Drivers ...")`

File: scripts/workers/rl_submission_worker.py [hunk 3]
```diff
@@ -247,6 +250,27 @@ def update_failed_jobs_and_send_logs(
...
+    logging.info("Using daemonset file: %s", link)
+    f = urllib.urlopen(link)
+    daemonset_spec = yaml.load(f, yaml.FullLoader)
+    ext_client = client.ExtensionsV1beta1Api(api_instance)
+    try:
+        namespace = daemonset_spec["metadata"]["namespace"]
+        ext_client.create_namespaced_daemon_set(namespace, daemonset_spec)
+    except ApiException as e:
+        if e.status == 409:
+            logging.info("GPU driver daemon set has already been installed")
```
Comment (RishabhJain2018) [inline scripts/workers/rl_submission_worker.py:269]: `logging.info("GPU driver daemon set has already been installed")` -> `logging.info("Nvidia GPU driver daemon set has already been installed")`

File: scripts/workers/rl_submission_worker.py (hunk 1 above)
Comment (Ayukha) [reply scripts/workers/rl_submission_worker.py:261]: Sure

**Comment by Ayukha** [comment]
@RishabhJain2018 Updated!

**Pull request merged**